examples/shared/src/test/scala/com/example/function/mockitostyle/HigherOrderFunctionsTest.scala

```diff
@@ -21,9 +21,9 @@
 package com.example.function.mockitostyle
 
 import org.scalamock.scalatest.MockFactory
-import org.scalatest.FreeSpec
+import org.scalatest.freespec.AnyFreeSpec
```

Comment:
if we adopt this new `AnyFreSpec` with ScalaTest 3.1, do users of ScalaMock also need to change their imports?

Reply:
They can, but they will see deprecated warnings. Please read the [release notes](http://www.scalatest.org/release_notes/3.1.0) for ScalaTest for more information. Thank you @barkhorn

Reply:
ty for the explanation :)